Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -201,9 +201,15 @@ import BraintreeCore
             }
         }
     }
+    
+    /// Indicates whether the Venmo App is installed.
+    /// - Warning: This method is currently in beta and may change or be removed in future releases.
+    public func isVenmoAppInstalled() -> Bool {
+        application.isVenmoAppInstalled()
+    }
 
     /// Returns true if the proper Venmo app is installed and configured correctly, returns false otherwise.
-    @objc public func isVenmoAppInstalled() -> Bool {
+    @objc public func isVenmoAppSwitchAvailable() -> Bool {
```

Comment:
Was just looking at this on the Android side and I think it makes more sense to keep these separated in the `ShopperInsightsClient` - otherwise we need to do refactoring with the current use of app installed methods in the PayPal module to make them public (or duplicate code).

Reply:
Cool, totally agree. We can have Sammy update the specs in our wiki pages once she's back. Sorry to change this from what was in the ticket @stechiu - but lets move both of the app installed methods and tests to the Shopper Insights module.

Reply:
I've moved them to ShopperInsightsClient